cmake/BLTMacros.cmake

```diff
@@ -1548,8 +1539,18 @@ macro(blt_convert_to_system_includes)
        message(FATAL_ERROR "TARGET is a required parameter for the blt_convert_to_system_includes macro.")
     endif()
 
```

Comment:
Should we add a check here to make sure that the target is not `IMPORTED`?

In https://github.com/LLNL/axom/pull/1017/commits/32ce9f28e95e847d553d08a0ad8a076917e17313, I had a workaround based on an error about that.

I suspect it's not necessary since I converted the code there to `blt_convert_to_system_includes` and no longer saw the error, but wanted to double check with you.

Reply:
My guess is the `target_include_directories` was causing that error. I say leave it until it causes another error.